### ISSUE TITLE
A new PRV segment Mapper after the HI segment

### ISSUE THREAD

**Issue by nikhil-javalkar** [imsweb/x12-parser#64]
As per our inputs file we need a PRV segment mapper after our HI segment mapper in the mapping/837.5010.X222.A1.xml file. Let us know if we can add our own Mapping in the file and contribute ourselves.

Thanks,
Nikhil Javalkar

**Comment by angelaszek** [comment]
Hello, I submitted a new issue (#65) that will allow custom mapping files to be passed to the parser to accommodate specs that differ from the supported standards. We will be working on it as time is available. I believe this will help address this issue?

**Comment by nikhil-javalkar** [comment]
Hi @ctmay4 could you please help me with the access to this repository so that i can create a pull request.

**Comment by nikhil-javalkar** [comment]
Hi @ctmay4 Could you please help us with the access to the repository to create a pull request.

**Comment by nikhil-javalkar** [comment]
Also @ctmay4 Can we just add the mapping for the missing segment in the desired mapping file and create a pull request for the same just to suffice the requirement for now ?

**Comment by angelaszek** [comment]
I don't believe a PRV segment after an HI segment is standard so it's not something we want to include as part of this library. You are welcome to make this change on a local version that you maintain though.

**Comment by nikhil-javalkar** [comment]
[SampleInputFile.txt](https://github.com/imsweb/x12-parser/files/8668063/SampleInputFile.txt)

Hi @ctmay4 Attached is the sample file we are getting as an input. It has a PRV segment after HI segment which is currently missing in the mapper file.

**Comment by angelaszek** [comment]
This will be in issue #65.
